src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let sumIsShowed = 0;
```

Comment:
i think it can be boolean value instead of 1, 0

Reply:
Good point!))